### ISSUE TITLE
Three-finger tap interferes with the Zoom Accessibility feature of iPhone/iPad, hence does not work

### ISSUE THREAD

**Issue by nikolay-diyanov** [NativeScript/ios-runtime#132]
If you have the Accessibility --> Zoom feature of your iDevice enabled, the three-finger tap gesture will not work to LiveSync the iOS app.

**Comment by PanayotCankov** [comment]
This is now outside our domain.
